src/components/task-plan/homework.cjsx

```diff
@@ -219,9 +217,13 @@ SectionTopic = React.createClass
   render: ->
     classes = ['section']
     classes.push('selected') if @props.active
+    isChecked = 'checked' if @props.active
     classes = classes.join(' ')
 
     <div key={@props.section.id} className={classes} onClick={@toggleSection}>
+      <span className="-section-checkbox">
+        <input type="checkbox" checked={isChecked}/>
```

Comment:
Would `<label htmlFor="...">` instead of the parent `div` simplify the `e.stopPropagation` lines that were added below?